{"uid":"a5f0ce6a1a9a9def","name":"Sign up with invalid data","fullName":"vizsgaremek.test.test_conduit.TestSignUp#test_sign_up_with_csv_invalid","historyId":"db178a6f3c6e588115a95d6365ccc185","time":{"start":1682228476409,"stop":1682228476409,"duration":0},"status":"broken","statusMessage":"FileNotFoundError: [Errno 2] No such file or directory: 'datasource_users.csv'","statusTrace":"self = <test_conduit.TestSignUp object at 0x7f4a9c1d81f0>\n\n    @allure.id(\"TC-02\")\n    @allure.title(\"Sign up with invalid data\")\n    def test_sign_up_with_csv_invalid(self):\n        wait = WebDriverWait(self.browser, 5)\n>       for user in get_users_from_file():\n\nvizsgaremek/test/test_conduit.py:48: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\n    def get_users_from_file():\n        #with open('vizsgaremek/test/datasource_users.csv', 'r', encoding='UTF-8') as datafile:\n>       with open('datasource_users.csv', 'r', encoding='UTF-8') as datafile:\nE       FileNotFoundError: [Errno 2] No such file or directory: 'datasource_users.csv'\n\nvizsgaremek/test/main_functions.py:35: FileNotFoundError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_xunit_setup_method_fixture_TestSignUp","time":{"start":1682228475495,"stop":1682228476408,"duration":913},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"afterStages":[{"name":"_xunit_setup_method_fixture_TestSignUp::0","time":{"start":1682228476419,"stop":1682228476475,"duration":56},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"labels":[{"name":"as_id","value":"TC-02"},{"name":"parentSuite","value":"vizsgaremek.test"},{"name":"suite","value":"test_conduit"},{"name":"subSuite","value":"TestSignUp"},{"name":"host","value":"fv-az1030-985"},{"name":"thread","value":"3147-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"vizsgaremek.test.test_conduit"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"a5f0ce6a1a9a9def.json","parameterValues":[]}